### DISCUSSION TITLE
How do I get a PDB of the combined antigen/antibody complex from Smina?

### DISCUSSION THREAD

**Discussion by tony-res** [martinez-zacharya/TRILL#29] [Q&A] [answered]
I was able to run an antibody/antigen docking with Smina using TRILL. I'm not sure if it outputs a PDB of the combined antigen-antibody in the best docking pose. Does it? If so, where can I find that?

Thanks.
-Tony

**Comment by martinez-zacharya** [comment, marked as answer]
Hi Tony, sorry for the delay! I've been doing some refactoring of TRILL and doing some last-minute bug fixes for the preprint coming out. I would also like to apologize about the misinformation, I don't think smina is able to dock proteins to proteins properly, you should go ahead with using lightdock for now. I'm in the process of updating the documentation and hopefully adding a new method for protein-protein docking, but I would avoid using smina for this purpose.

**Reply by tony-res** [reply]
Thanks!
